ui/app/models/pki/pki-role-engine.js

```diff
@@ -13,6 +13,7 @@ const validations = {
 export default class PkiRoleEngineModel extends Model {
   @attr('string', { readOnly: true }) backend;
 
+  /* Overriding default options */
```

Comment:
Yep, because I'm using OpenAPI I'm overriding the values returned there. I can add Overriding API defaults?

Reply:
Yeah - I think something like `Overriding Attrs from OpenAPI`. I think what was unclear to me, since I don't have much experience with OpenApi, was I didn't realize that _all_ the attrs are defined there, that we're just intercepting some of them.